packages/contracts-bedrock/test/OptimismPortal.t.sol

```diff
@@ -898,15 +896,17 @@ contract OptimismPortalUpgradeable_Test is CommonTest {
 
     /// @dev Tests that the proxy cannot be initialized twice.
     function test_initialize_cannotInitProxy_reverts() external {
+        SuperchainConfig superchainConfig = SuperchainConfig(mustGetAddress("SuperchainConfig"));
         vm.expectRevert("Initializable: contract is already initialized");
-        optimismPortal.initialize({ _paused: false });
+        optimismPortal.initialize(superchainConfig);
     }
 
     /// @dev Tests that the implementation cannot be initialized twice.
     function test_initialize_cannotInitImpl_reverts() external {
         address opImpl = mustGetAddress("OptimismPortal");
+        SuperchainConfig superchainConfig = SuperchainConfig(mustGetAddress("SuperchainConfig"));
         vm.expectRevert("Initializable: contract is already initialized");
-        OptimismPortal(payable(opImpl)).initialize({ _paused: false });
+        OptimismPortal(payable(opImpl)).initialize(superchainConfig);
```

Comment:
These tests are technically redundant with the post deploy checks and Initializable.t.sol, but I've kept them here for now to simplify the refactoring. 

Probably should remove IMO, wdyt?